internal/integrations/utils.go

```diff
@@ -0,0 +1,194 @@
+package integrations
+
+import (
+	"fmt"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
```

Comment:
```suggestion
	"gopkg.in/yaml.v3"
```

Reply:
v2 is already used across the code, I did not want to update a dependency we do not own or import two versions of yaml